src/main/java/Backend/FileDownloader.java

```diff
@@ -143,6 +143,8 @@ private static void downloadFile() {
 
             } catch (SecurityException e) {
                 messageBroker.sendMessage("Write access to " + dir + fileName + " denied !", LOGGER_ERROR, "download");
+            } catch (FileNotFoundException fileNotFoundException) {
+                messageBroker.sendMessage(FILENOTFOUND, LOGGER_ERROR, "download");
```

Comment:
Provides more details to the error message when the file is not found instead of a simple message of failed to download contents.